examples/CustomEntitlementComputationSample/app/src/main/java/com/revenuecat/sample/MainApplication.kt

```diff
@@ -0,0 +1,33 @@
+package com.revenuecat.sample
+
+import android.app.Application
+import com.revenuecat.purchases.LogLevel
+import com.revenuecat.purchases.Purchases
+import com.revenuecat.purchases.PurchasesConfiguration
+import com.revenuecat.sample.data.Constants
+
+class MainApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        /*
+        Enable debug logs before calling `configure`.
+         */
+        Purchases.logLevel = LogLevel.DEBUG
+
+        /*
+        Initialize the RevenueCat Purchases SDK.
+
+        - appUserID is nil, so an anonymous ID will be generated automatically by the Purchases SDK. Read more about Identifying Users here: https://docs.revenuecat.com/docs/user-ids
```

Comment:
Maybe add docs that we should pass the application context? (as opposed to an activity context)